client/api/go-client/client_test.go

```diff
@@ -19,13 +19,13 @@ import (
 	"reflect"
 	"testing"
 
-	"github.com/codegangsta/negroni"
 	"github.com/gorilla/mux"
 	"github.com/heketi/heketi/apps/glusterfs"
 	"github.com/heketi/heketi/middleware"
 	"github.com/heketi/heketi/pkg/glusterfs/api"
 	"github.com/heketi/heketi/pkg/utils"
 	"github.com/heketi/tests"
+	"github.com/urfave/negroni"
```

Comment:
I think this and all the vendor/ updates should be in separate patches. (not necessarily separate PR, but separate patch for sure.) We should try to avoid mixing our own code changes with vendor/ updates)